src/main/java/teammates/logic/backdoor/BackDoorLogic.java

```diff
@@ -152,9 +152,9 @@ public String persistDataBundle(DataBundle dataBundle)
             response = injectRealIds(response);
         }
         frDb.createFeedbackResponses(responses.values());
-        
-        for(FeedbackSessionAttributes session : sessions.values()){
-            updateRespondants(session.feedbackSessionName, session.courseId);
+
+        for (FeedbackResponseAttributes response : responses.values()) {
+            updateRespondants(response.feedbackSessionName, response.courseId);
```

Comment:
The new backdoor operation that I added (`createFeedbackResponse`) takes in `FeedbackResponseAttributes` as the only parameter. I modified this file so that `BackDoorLogic` requires only `FeedbackResponseAttributes` to create a feedback response (previously `BackDoorLogic` requires both `FeedbackSessionAttributes` and `FeedbackResponseAttributes`).